packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -53,13 +54,27 @@ import {
   useDestinationAddressStore
 } from './AdvancedSettings'
 import { USDCDepositConfirmationDialog } from './USDCDeposit/USDCDepositConfirmationDialog'
+import { USDCWithdrawalConfirmationDialog } from './USDCWithdrawal/USDCWithdrawalConfirmationDialog'
 
 const onTxError = (error: any) => {
   if (error.code !== 'ACTION_REJECTED') {
     Sentry.captureException(error)
   }
 }
 
+type USDCWithdrawalConfirmationDialogStore = {
```

Comment:
Thinking of having `USDCWithdrawalConfirmationDialog's` state logic within `USDCWithdrawalConfirmationDialogStore.tsx` instead of this file for cleaner abstraction.
`TransferPanel.tsx` is already too big!

Reply:
yea i agree that `TransferPanel.tsx` is too long
i'd like to hear what everyone thinks about this one
not really sure where best to put it
`USDCWithdrawalConfirmationDialog.tsx` seems to be a good place but it's not used in `USDCWithdrawalConfirmationDialog.tsx` at all

Reply:
As per our discussion - We will refactor the `useDialog` hook in a new PR